compiler/rustc_typeck/src/check/cast.rs

```diff
@@ -168,7 +169,7 @@ pub enum CastError {
 }
 
 impl From<ErrorReported> for CastError {
-    fn from(ErrorReported: ErrorReported) -> Self {
+    fn from(_: ErrorReported) -> Self {
         CastError::ErrorReported
```

Comment:
Is there no use to `CastError::ErrorReported` containing a `ErrorReported`?

Reply:
Apparently not, at the moment...